pkg/cmd/project/delete/delete.go

```diff
@@ -0,0 +1,101 @@
+package delete
+
+import (
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/OctopusDeploy/cli/pkg/question"
+	"github.com/OctopusDeploy/cli/pkg/question/selectors"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/client"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/projects"
+	"github.com/spf13/cobra"
+	"io"
+)
+
+type DeleteOptions struct {
+	Client   *client.Client
+	Ask      question.Asker
+	Host     string
+	Writer   io.Writer
+	NoPrompt bool
+	idOrName string
```

Comment:
Is there any reason why `idOrName` is a private variable where the others are public?

(struct variables that start with lowercase letters are package-private in go. Uppercase are public.)

Reply:
no reason, I might have copied it from somewhere else, still getting my muscle memory in tune. Fixed.